src/components/alert/alert.stories.mdx

```diff
@@ -65,6 +65,29 @@ Alerts work well even when displayed at full width. Their contents will constrai
   </Story>
 </Canvas>
 
+<!-- TODO: Right now the icon needs to be set to the sad emoji.
+// As of right now, I don't know how to automatically add the icon on it's own.
+// That is a created issue but not a launch blocker.
+ -->
+
+## Error
+
+The class `c-alert__error` will override the orange default color and make the icon and background red. As of right now, the icon needs to be set to `sad-emoji`. An issue has been created to add it automatically.
+
```

Comment:
Can we give the icon a name like `face-confused` to align with [a similar unicode symbol](https://emojipedia.org/confused-face/) and to make things easy if we ever want to add more facial expressions.